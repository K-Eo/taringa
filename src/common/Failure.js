import React from 'react'

const Failure = () => {
  return (
    <div className="text-center text-danger">
      <p>¡Ratas! Algo salío mal. Intenta recargar la página</p>
    </div>
  )
}

export default Failure
