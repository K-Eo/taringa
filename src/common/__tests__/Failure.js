import React from 'react'

import Failure from '../Failure'

describe('Failure', () => {
  it('exists', () => {
    expect(Failure).toBeDefined()
  })

  it('renders', () => {
    const wrapper = shallow(<Failure />)

    expect(wrapper).toMatchSnapshot()
  })
})
