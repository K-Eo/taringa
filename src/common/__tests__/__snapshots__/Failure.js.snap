// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Failure renders 1`] = `
<div
  className="text-center text-danger"
>
  <p>
    ¡Ratas! Algo salío mal. Intenta recargar la página
  </p>
</div>
`;
